compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -1884,12 +1936,13 @@ fn describe_enum_variant(
     let (offsets, args) = if use_enum_fallback(cx) {
         // If this is not a univariant enum, there is also the discriminant field.
         let (discr_offset, discr_arg) = match discriminant_info {
-            RegularTag { tag_field, .. } => {
+            Some(DirectTag { tag_field, .. }) => {
                 // We have the layout of an enum variant, we need the layout of the outer enum
                 let enum_layout = cx.layout_of(layout.ty);
                 let offset = enum_layout.fields.offset(tag_field.as_usize());
-                let args =
-                    ("RUST$ENUM$DISR".to_owned(), enum_layout.field(cx, tag_field.as_usize()).ty);
+                let tag_name =
+                    if cx.tcx.sess.target.is_like_msvc { "variant$" } else { "RUST$ENUM$DISR" };
```

Comment:
This condition will always be true, because `use_enum_fallback` is defined like this:
```rust
fn use_enum_fallback(cx: &CodegenCx<'_, '_>) -> bool {
    cx.sess().target.is_like_msvc
}
```

Reply:
Good catch, thanks!